### PR TITLE
Enrolment summary uses only enrolments which occurrence is in future.

### PR DESCRIPTION
**Pull request by nikomakela**
PT-1364 - Enrolment summary emails fails sending because they couldn't fetch events from LinkedEvents. I found out  that summary tries to fetch all pending enrolments from history - to the beginning of time. LinkedEvents deletes events from past.

Also did some refactoring related to the notification sending: Moved the functions where I think they belong, since QuerySet should not send any emails.

### CODE REVIEW

**Comment by nikomakela** [comment]
NOTE: The last commit contains the actual changes needed. Others are for refactoring.

File: occurrences/models.py [hunk 1]
```diff
@@ -472,60 +471,34 @@ def save(self, *args, **kwargs):
...
+    def pending_enrolments_by_email(self, email: str):
+        return self.filter(
+            occurrence__p_event__contact_email=email, status=Enrolment.STATUS_PENDING,
+        )
 
-    def get_by_unique_id(self, unique_id):
-        compound_id = get_node_id_from_global_id(unique_id, "EnrolmentNode")
-        enrolment_id, ts = compound_id.split("_")
-        return self.get(id=enrolment_id, enrolment_time=ts)
+    def new_enrolments_by_email(self, email: str):
```
Comment (charn) [inline occurrences/models.py:498]: or `approved_enrolments_by_email` ?

Comment (nikomakela) [reply occurrences/models.py:498]: Absolutely! I'll change it

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Pull request merged**